test/helpers.lua

```diff
@@ -88,8 +88,10 @@ function module.pcall_err(fn, ...)
   if status == true then
     error('expected failure, but got success')
   end
-  local errmsg = tostring(rv):gsub('^[^:]+:%d+: ', '')
-  return errmsg
+  local errmsg = tostring(rv):gsub('^[a-zA-Z]?:?[^:]+:%d+: ', '')
+  errmsg = tostring(errmsg):gsub('^Error executing lua: ', '')
+  errmsg = tostring(errmsg):gsub('^Error loading lua: ', '')
```

Comment:
I think we want the tests to be explicit  . Not remove these prefixes

Reply:
uhh. So, should I use string.match instead of pcall_err for error messages like vim.shared that contain file names and line numbers?

Reply:
No, I mean only the `Error executing lua` and `Error loading lua` messages shouldn't be removed.